ui/app/templates/components/generated-item-list.hbs

```diff
@@ -5,16 +5,10 @@
 
 <PageHeader as |p|>
   <p.top>
-    <KeyValueHeader @path="vault.cluster.access.methods">
```

Comment:
I replaced `KeyValueHeader` components that yielded breadcrumbs, but left self-closing `KeyValueHeader` components and updated the hbs file